packages/pg-pubsub/src/PgGenericSubscriptionPlugin.ts

```diff
@@ -142,6 +174,12 @@ const PgGenericSubscriptionPlugin: Plugin = function (
               topic: {
                 type: new GraphQLNonNull(GraphQLString),
               },
+              immediate: {
```

Comment:
The word `immediate` here is a little confusing - without reading the docs (which don't appear in Explorer, for example), it feels like you should set it because you want to immediately listen ("Why wouldn't I want to listen immediately?!").

We need a name for this that more intuitively encompasses what it does. I'm not super happy with `initialEvent` because it feels like rather than being a boolean it should be an input object to specify the initial event (I do not want to do this!). But `includeInitialEvent`/`triggerInitialEvent`/`triggerEventOnStart`/`withStartEvent`/`withInitialEvent`/`eventOnInit`/etc feel too wordy. And `fireImmediately` feels a bit unfriendly.

Having spent a good 5 minutes pacing the living room trying to think of a better name, right now `initialEvent` is my preferred name for this. But I'm sure there's a better one out there.

Naming things :man_facepalming: 

```suggestion
              initialEvent: {
```